payment-request/payment-response/payerEmail-attribute-manual.https.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="https://w3c.github.io/payment-request/#dom-paymentresponse-payeremail">
+<title>
+  PaymentResponse.prototype.payerEmail attribute
+</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="helpers.js"></script>
+<h2>payerEmail attribute</h2>
+<p>
+  When requested, please use "wpt@w3.org" as the email.
+</p>
+<ol>
+  <li>
+    <button onclick="runManualTest(this, { requestPayerEmail: undefined }, { payerEmail: null })">
+      payerEmail attribute is null when requestPayerEmail is undefined.
+    </button>
+  </li>
```

Comment:
Maybe we also can have a test for null PaymentOption, the default option set by browser can be tested. 
<button onclick="runManualTest(this, null, { payerEmail: null })">
  payerEmail attribute is null when PaymentOptions is set by browser.
</button>